mod/b_extended_profile/views/default/profile/wrapper.php

```diff
@@ -81,26 +85,27 @@
             finit_ajax_block('work-experience');
         }
 
-        //if(elgg_is_logged_in()){
-            if ( has_content($user, 'gc_skills') ) {
-                init_ajax_block(elgg_echo('gcconnex_profile:gc_skills'), 'skills', $user);
-                echo elgg_view('b_extended_profile/skills');
-                finit_ajax_block('skills');
-            }
-        //}
+        if ( has_content($user, 'gc_skills') && strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') === false ) {
```

Comment:
There is apparently [`stristr`](http://php.net/manual/en/function.stristr.php) for the case insensitive version.

Reply:
You fixed all of them, but the one I commented on :stuck_out_tongue: